source/core/parameters_lagrangian.cc

```diff
@@ -212,9 +216,12 @@ namespace Parameters
                             rolling_friction_coefficient_particle,
                             surface_energy_particle);
 
-      g[0] = prm.get_double("gx");
-      g[1] = prm.get_double("gy");
-      g[2] = prm.get_double("gz");
+      // Depreciated parameter handling
+      // <g> used to be 3 parameters: <gx>, <gy> and <gz>
+      // If <gx> is in the input file, it will be used as the value for <g>
+      // as an alias. This way, the parameter <g> is not a tensor and allows the
+      // parsing of depreciated parameters.
```

Comment:
```suggestion
      // Deprecated parameter handling
      // <g> used to be 3 parameters: <gx>, <gy> and <gz>
      // If <gx> is in the input file, it will be used as the value for <g>
      // as an alias. This way, the parameter <g> is not a tensor and allows the
      // parsing of depreciated parameters.
```